src/commcare_cloud/environment/main.py

```diff
@@ -267,6 +267,7 @@ def create_generated_yml(self):
             'dev_users': self.users_config.dev_users.to_json(),
             'authorized_keys_dir': '{}/'.format(self.paths.authorized_keys_dir),
             'known_hosts_file': self.paths.known_hosts,
+            'commcarehq_repository': self.fab_settings_config.code_repo,
```

Comment:
where does this get used?

Reply:
When ansible is first pulling down the code, it uses `commcarehq_repository` as the git endpoint. Not sure if it makes a real difference, but just seemed more correct.